crates/services/p2p/src/service.rs

```diff
@@ -339,7 +383,11 @@ pub struct Task<P, V, B> {
     request_sender: mpsc::Sender<TaskRequest>,
     database_processor: HeavyTaskProcessor,
     broadcast: B,
+    #[allow(dead_code)]
+    tx_pool: T,
     max_headers_per_request: usize,
+    #[allow(dead_code)]
```

Comment:
```suggestion
    tx_pool: T,
    max_headers_per_request: usize,
```

Why do we need dead code here?

Reply:
Outdated spec. Removed.